### PR TITLE
Add build script for lz4-java

### PR DESCRIPTION
**Pull request by Abhijit-Mane**
Build script for https://github.com/lz4/lz4-java

Signed-off-by: Abhijit Mane <abhijman@in.ibm.com>

### CODE REVIEW

**Review by krishvoor — COMMENTED** [review]
/lgtm

File: l/lz4-java/build.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# clone
+git clone $BRANCH https://github.com/lz4/lz4-java.git || (echo "git clone failed"; exit $?)
+cd lz4-java
+git submodule init
+git submodule update
+ant ivy-bootstrap
+ant 
+
+# copy build artifacts
+cp -f ./dist/lz4-java-1.6-SNAPSHOT.jar ../
```
Comment (afsanjar) [inline l/lz4-java/build.sh:41]: 1. should SNAPSHOT tag be removed after if the build is not from the master?
2. Should this be "cp -f ./dist/lz4-java-*-SNAPSHOT.jar ../"

**Review by afsanjar — COMMENTED** [review]
Look at my comment.

**Comment by Abhijit-Mane** [comment]
Closing this PR-950. Submitted new one.

**Comment by Abhijit-Mane** [comment]
Closing this PR-950. Submitted new one.

**Pull request closed without merging**